salt/utils/gitfs.py

```diff
@@ -1005,6 +1038,17 @@ def env_is_exposed(self, tgt_env):
         Check if an environment is exposed by comparing it against a whitelist
         and blacklist.
         """
+        if self.opts.get("__git_pillar", False) is True:
```

Comment:
using truthiness is preferred
```suggestion
        if self.opts.get("__git_pillar", False):
```